sections/collage.liquid

```diff
@@ -186,7 +186,10 @@
                     </button>
                   </div>
                 </modal-opener>
-                <modal-dialog id="PopupModal-{{ block.id }}" class="modal-video media-modal color-background-1">
+                <modal-dialog
+                  id="PopupModal-{{ block.id }}"
+                  class="modal-video media-modal color-{{ settings.color_schemes | first }}"
```

Comment:
This is an example of where I'm dynamically referencing the "first" (and fallback) scheme rather than "background-1" specifically.

Reply:
Do you need this `color` class at all now? Won't the variables for the first scheme be defined in `:root`?